### ISSUE TITLE
cqf-ruler-dstu3 UI endpoint gives 404

### ISSUE THREAD

**Issue by losborne** [DBCG/cqf-ruler#158]
Using either of these tags of the `contentgroup/cqf-ruler` docker image:
![image](https://user-images.githubusercontent.com/14879344/73019593-e5cc3700-3df1-11ea-8358-86a27c48f361.png)

When we navigate to the endpoint http://localhost:8080/cqf-ruler-dstu3/ we see this error:
```
2020-01-23 20:08:12.520 [qtp1375995437-16] WARN  o.s.web.servlet.PageNotFound [DispatcherServlet.java:1249] No mapping for GET /cqf-ruler-dstu3/
```

![image](https://user-images.githubusercontent.com/14879344/73019474-a998d680-3df1-11ea-944b-f96bcebaea08.png)

The API calls using the endpoint `http://localhost:8080/cqf-ruler-dstu3/fhir` seem to still be functioning, but the UI for dstu3 is not loading.  The r4 endpoint does load.

**Comment by p9g** [comment]
look at https://github.com/DBCG/cqf-ruler/commit/124f861df788876f1db3a2e33a1f56927087584f#diff-ac1c5e2e770ce1583ef97fb801d0c53bR28 

typo messed up `</servlet-class>`

**Comment by JPercival** [comment]
This was due to an oversight in the most recent refactor / release. We intend to re-enable the testing endpoint shortly.

**Comment by JPercival** [comment]
Resolved in #168
